src/index.html

```diff
@@ -1,13 +1,384 @@
+          </div>
+
+          <div class="about-us__statistic">
+
+            <div class="about-us__statistic--first-column">
+              <p class="big-text about-us__big-text">2015</p>
+              <p class="body-text about-us__body-text">Foundation year</p>
+            </div>
+
+            <div class="about-us__statistic--second--column">
+              <p class="big-text about-us__big-text">23</p>
+              <p class="body-text about-us__body-text">Stores</p>
+            </div>
+
+            <div class="about-us__statistic--third-column">
+              <p class="big-text about-us__big-text">155</p>
+              <p class="body-text about-us__body-text">Companies</p>
+            </div>
+
+            <div class="about-us__statistic--forth-column">
+              <p class="big-text about-us__big-text">10k+</p>
+              <p class="body-text about-us__body-text">Clients</p>
+            </div>
+
+          </div>
+        </section>
+
+      </div>
+
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__content">
+
+          <div class="footer__menu">
+            <a href="#" class="footer__logo__link">
+              <img
+                src="./images/footer/logo.svg"
+                alt="logo"
+                class="footer__logo"
+              >
+            </a>
+
+            <nav class="footer__nav nav">
+              <p class="uppercase footer__text">Menu</p>
+              <ul class="footer__nav--list">
+                <li class="footer__nav--item">
+                  <a href="#" class="footer__link">Products</a>
+                </li>
+                <li class="footer__nav--item">
+                  <a href="#" class="footer__link">About Us</a>
+                </li>
+                <li class="footer__nav--item">
+                  <a href="#" class="footer__link">Blog</a>
+                </li>
+                <li class="footer__nav--item">
+                  <a href="#" class="footer__link">Terms & Privacy</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+
+          <div class="footer__contacts">
+
+            <div class="footer__contacts--wrapper">
+
+              <h6 class="footer__title" id="contact-us">Contact us</h6>
+
+              <div class="footer__contacts--data">
+
+                <div class="footer__contacts--data--to-call">
+                  <p class="uppercase footer__text">Call us</p>
+                  <div>
+                    <a href="tel:(629) 555-0129" class="body-text footer__link">(629) 555-0129</a>
+                  </div>
+                  <div>
+                    <a href="tel:(603) 555-0123" class="body-text footer__link">(603) 555-0123</a>
+                  </div>
+                  <div>
+                    <a href="tel:(704) 555-0127" class="body-text footer__link">(704) 555-0127</a>
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
                    <a href="tel:7045550127" class="body-text footer__link">(704) 555-0127</a>
```